packages/clerk-js/src/ui/components/SignIn/SignInFactorOnePasswordCard.tsx

```diff
@@ -83,6 +86,14 @@ export const SignInFactorOnePasswordCard = (props: SignInFactorOnePasswordProps)
           return clerk.__internal_navigateWithError('..', err.errors[0]);
         }
 
+        if (isPasswordPwnedError(err)) {
+          if (onPasswordPwned) {
+            card.setError({ ...err.errors[0], code: err.errors[0].code + '__sign_in' });
```

Comment:
Need to add a suffix to target the l10n key we want.

Reply:
I'd say just hardcode it here since the type will always be the same.